src/components/current-task/CurrentTask.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Task from '../global/Task/Task';
+import CurrentTaskNotes from './CurrentTaskNotes';
+
+function CurrentTask() {
+	return (
+		<div className='current-task-container'>
```

Comment:
Not wanting you to change all your classnames, but you should have a look into BEM naming. Combine this with SCSS nested styles, and stylesheets can become a lot easier to read